packages/main/src/telegram_webhook.ts

```diff
@@ -11,7 +11,7 @@ export default class TelegramWebhook extends Webhook {
     sha256(this.token).then((access_key) =>
       fetch_json(
         addSearchParams(new URL(`${this.api.href}/setWebhook`), {
-          url: `${this.url.href}/${access_key}`,
+          url: `${this.url.href}${access_key}`,
           max_connections: "100",
           allowed_updates: JSON.stringify(["message", "inline_query"]),
```

Comment:
@codebam this is the call to Telegram for registering the webhook, right?
Without my change this registered the webhook as `https://my-host.com//sha256` (notice the double slash) when calling `https://my-host.com/sha256?command=set`. That's cause the webhook to not be accepted by the bot and my commit fixes this. 
I'm not sure how this works for you without this. Are you registering the webhook differently?

Reply:
Access key checking isn't even done in Webhook. It's done in Handler here https://github.com/codebam/cf-workers-telegram-bot/blob/master/packages/main/src/handler.ts#L80

Reply:
my comment wasn't about the access key checking. Your readme says
```
Example URL for setting the Webhook and dropping pending updates:

https://cf-workers-telegram-bot.codebam.workers.dev/a948904f2f0f479b8f8197694b30184b0d2ed1c1cd2a1ec0fb85d299a192a447?command=set
```
Which is exactly what I did (with my own sha of course). Then I called `command=get` and the webhook URL was shown as with double slash. And in fact the bot didn't work. Then I made my change and it did work.

I don't get why you're so dismissive, but whatever, that's my last attempt

Reply:
Does your PR make it work on your repo?

Reply:
it does

Reply:
https://github.com/codebam/cf-workers-telegram-bot/commit/920fffc91821728bba53bb4ebe795003d4bd5fef

I added a regex for this as I couldn't reproduce it on my end all the time